### ISSUE TITLE
login failed

### ISSUE THREAD

**Issue by liontiger** [KWAc/gmailClicker#1]
![gmail error](https://cloud.githubusercontent.com/assets/13665393/9094389/1f4565ee-3bcf-11e5-8297-6111a575ed51.jpg)

any thing wrong ? username and password is correct,indeed i received mail from gmail security stating that unknown app was trying to sign in ..

**Comment by KWAc** [comment]
Yes you must disable some security options in gmail account security
